Subforms/FilterSettings.Designer.cs

```diff
@@ -181,6 +182,16 @@ private void InitializeComponent()
             this.ShinyCheck.TabIndex = 6;
             this.ShinyCheck.Text = "Shiny?";
             this.ShinyCheck.UseVisualStyleBackColor = true;
+            //
+            // ECCheck
+            //
+            this.ECCheck.AutoSize = true;
+            this.ECCheck.Location = new System.Drawing.Point(104, 285);
+            this.ECCheck.Name = "ECCheck";
+            this.ECCheck.Size = new System.Drawing.Size(60,19);
+            this.ECCheck.TabIndex = 19;
+            this.ECCheck.Text = "EC%100?";
```

Comment:
```suggestion
            this.ECCheck.Text = "EC % 100 = 0?";
```

`EC%100?` doesn't really tell the user anything, EC%100 what?
It might also be pertinent to add a tooltip explaining that it'll search for Tandemaus that will evolve into 3fam and Dun into 3-seg, but worded better

Reply:
Sure thing, I'll get those updated